Makefile

```diff
@@ -26,7 +26,9 @@ clean:
 	rm -rf $(DESTDIR)
 
 run:
-	go run cmd/main.go
+	cd $(SRC_DIR) && \
+	AZTUI_CONFIG_PATH=$(PWD)/conf/default.yaml go run cmd/main.go && \
+	cd ..
```

Comment:
Did this work before? Because we were not **cd**ing into **src**.

Reply:
It worked before I moved all the src up into the the src/ subdirectory (in pursuit of making RPM packaging easier).